lib/ansible/modules/cloud/hcloud/hcloud_server_network.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Hetzner Cloud GmbH <info@hetzner-cloud.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = """
+---
+module: hcloud_server_network
+
+short_description: Manage the relationship between Networks and servers
+
+version_added: "2.9"
+
+description:
+    - Create, update and delete cloud subnetworks on the Hetzner Cloud.
+
+author:
+    - Lukas Kaemmerling (@lkaemmerling)
+
+options:
+    network:
+        description:
+            - The name of the Hetzner Cloud Networks.
+
+        type: str
+        required: true
+    server:
+        description:
+            - The name of the Hetzner Cloud server.
+        type: str
+        required: true
+    ip:
+        description:
+            - The IP the server should have.
+        type: str
+    alias_ips:
+        description:
+            - Alias IPs the Server has.
+        type: list
+    state:
+        description:
+            - State of the subnetwork.
+        default: present
+        choices: [ absent, present ]
+        type: str
+
+requirements:
+  - hcloud-python >= 1.3.0
+
+extends_documentation_fragment: hcloud
+"""
+
+EXAMPLES = """
+- name: Create a basic server network
+  hcloud_server_network:
+    network: my-network
+    server: my-server
+    state: present
+
+- name: Create a server network and specify the ip address
+  hcloud_server_network:
+    network: my-network
+    server: my-server
+    ip: 10.0.0.1
+    state: present
+
+- name: Create a server network and add alias ips
+  hcloud_server_network:
+    network: my-network
+    server: my-server
+    ip: 10.0.0.1
+    alias_ips:
+       - 10.1.0.1
+       - 10.2.0.1
+    state: present
+
+- name: Ensure the server network is absent (remove if needed)
+  hcloud_server_network:
+    network: my-network
+    server: my-server
+    state: absent
+"""
+
+RETURN = """
+hcloud_server_network:
+    description: The relationship between a server and a network
+    returned: always
+    type: complex
+    contains:
+        network:
+            description: Name of the Network
```

Comment:
s/Network/network

Reply:
The product is called `Network` so we would leave it as it is.